packages/react/src/components/fieldset-legend/fieldset-legend.tsx

```diff
@@ -0,0 +1,14 @@
+import { c, classy } from '@onfido/castor';
+import React from 'react';
+
+/**
+ * Intended to be used within `Fieldset` component, providing a text legend.
```

Comment:
💅🏻 

```suggestion
 * Use within `Fieldset` component to provide a legend to an input.
```

Reply:
Will not apply this, as I'd like to avoid using "input" here.

Reply:
No worries, maybe this then?

```suggestion
 * Use within `Fieldset` component to provide a text legend.
```